src/category_theory/monad/limits.lean

```diff
@@ -82,6 +83,86 @@ def forget_creates_limits (D : J ⥤ algebra T) [has_limit.{v₁} (D ⋙ forget
       end },
     uniq' := λ s m w, by { ext1, ext1, simpa using congr_arg algebra.hom.f (w j) } } }
 
+namespace forget_creates_colimits
+
+variables (D : J ⥤ algebra T) [has_colimit.{v₁} (D ⋙ forget T)]
+
+@[simps] def γ : ((D ⋙ forget T) ⋙ T) ⟶ (D ⋙ forget T) := { app := λ j, (D.obj j).a }
+
+@[simps]
+def c : cocone ((D ⋙ forget T) ⋙ T) :=
+{ X := colimit (D ⋙ forget T),
+  ι := γ D ≫ (colimit.cocone (D ⋙ forget T)).ι }
+
+variable [preserves_colimits_of_shape J T]
+
+@[reducible, nolint doc_blame]
+def lambda :=
```

Comment:
Could you please add the type?
I'm not sure if I like all the `nolint doc_blame` attributes. I'm not an expert on monads, and to me this file is very much write-only. I can't recognise which parts of the code have "math-content" and which words are there as glue, just to make Lean happy. So I would rather have docstrings that confirm that something does what I hope it does.
I know that writing docs is not the most pleasant thing out there. But we're fighting the battle.
(Large parts of the library were written before we had docstring rules in place. But in the last couple of months, I don't think `no_lint doc_blame`s have been added that much at all.

Reply:
I agree. Let's avoid `no_lint doc_blame`. I suggest writing `/-- (implementation) something-something-something -/ for implementation detail functions. Even if you just say in words what the def is doing, that's helpful. Ideally the mathlib naming scheme says that the name itself will be an abbreviation of the return type. The doc string can sometime just be a full English sentence expressing the return type.